tests/workspace/test_workflows.py

```diff
@@ -57,6 +58,58 @@ def mock_airlock_reporter():
     httpretty.reset()
 
 
+class MockRepoWorkflowReporter(jobs.RepoWorkflowReporter):
+    def __init__(self, location):
```

Comment:
This `__init__` shouldn't be required, it's just calling super on the parent class